### DISCUSSION TITLE
7TV Plugin

### DISCUSSION THREAD

**Discussion by mohamadtout** [ReVanced/revanced-discussions#342] [Question]
I watch most esports broadcasts on YouTube cz I hate twitch's UI. It would be really nice if you can integrate the 7tv emote plugin into the Revanced App.

**Comment by oSumAtrIX** [comment]
The emotes would only be visible for you.

**Reply by mohamadtout** [reply]
yeah ik that, it's how the plugin works...
It will add to the user experience, and it would be a really nice step up from the Vanced App

**Reply by oSumAtrIX** [reply]
Why do you need to have an emote only you can see?

**Reply by mohamadtout** [reply]
In a livestream where most chatters are using these emotes on their desktop browser, and where the streamer is encouraging people to get it, it adds to the user experience so much rather than just looking at the chat and seeing the text of the emotes

**Reply by TheJeterLP** [reply]
I don't see a reason for this. It's actually very unreasonable. Never even heard of that plugin, I guess that barely someone uses it.

**Reply by bogadana** [reply]
lol everyone uses this/alternative plugins

**Reply by TheJeterLP** [reply]
> lol everyone uses this/alternative plugins

Guess I live behind the moon then lol

**Reply by Hyzian** [reply]
> lol everyone uses this/alternative plugins

i second this, its really popular within the livestreaming community

**Reply by Keiishu** [reply]
I third this.
It would be really great to have a BTTV/FFZ/7TV integration, just like the TwitchMod app.
As Hyzian said, it's really popular within the livestreaming community.

**Reply by Sleepywalker69** [reply]
There is a fundamental misunderstanding here of how Twitch works in 2025. These emotes aren't 'only visible to me.' They are currently visible to everyone on desktop, the streamer, and the thousands of other viewers.

​Right now, mobile users are the only ones excluded. When the chat spams a 7TV emote, desktop users see an image, but mobile users just see raw text (e.g., CatJam). We aren't asking for a local customization; we are asking for the mobile app to be able to render the same visual language that the rest of the community is already using.

​Respectfully, just because you might not personally use these extensions or follow current Twitch culture doesn't mean the feature is 'unreasonable.' It is the industry standard for the platform. Dismissing a highly requested feature simply because of a lack of familiarity with the ecosystem is frustrating for those of us trying to help improve the app.

**Comment by KhalidSzyslak** [comment]
I think people don't understand how 7tv and bttv emotes works, those emotes aren't special symbols they're just text phrases like "KEKW" or "GG" even if you don't have the plugin urself when you type these symbols in chat people who have it will see them as emotes, so the patch will just have to translate these phrases to emotes so you can see them as well. a good example on how this can be achieved is the TwitchMod app which add the same functionality to the Twitch Android app.

**Comment by Sleepywalker69** [comment]
Bumping this thread because I’d really love to see this implemented. Currently, I’m stuck using the outdated BTTV mobile app just to see emotes. Adding support for 7TV, BTTV, and FFZ would essentially complete the user experience, as so many top channels rely on custom emotes that are currently invisible on the standard mobile app.
